src/main/java/browserview/ChromeDriverEx.java

```diff
@@ -50,9 +52,7 @@ public void get(String url, boolean isForceRefresh) throws WebDriverException {
             logger.info("Test loading page: " + url);
             testGet();
         } else {
-            if (!isForceRefresh && (driver.getCurrentUrl().equalsIgnoreCase(url) ||
-                driver.getCurrentUrl().equalsIgnoreCase(url + "/commits") ||
-                driver.getCurrentUrl().equalsIgnoreCase(url + "/files"))) {
+            if (!isForceRefresh && (driver.getCurrentUrl().equalsIgnoreCase(url))) {
```

Comment:
Hmm, I did trace all usages of `ChromeDriverEx.get` before doing this; spent about 3-4 days covering every usage; ran all tests; manually tested each case where this method is involved through the UI. 

It's also weird because you would expect `get` to simply perform `GET` with the refresh check; the way it was, you could never use it to explicitly access other PR tabs when the browser is already in the PR's main tab. Does this violate SRP?

If I keep this old way then I can't use `ChromeDriverEx.get` for switching tabs at all. Implementing a slightly different `get` method just to allow the `*/commits` and `*/files` urls through feels wrong.